docs/tutorials/custom/custom-base-dog.md

```diff
@@ -22,12 +22,9 @@ cost: 190
 
 The [base component type](/components/base/) is useful for controlling mobile robots because it gives users intuitive steering controls to use in code as well as from the [Viam app](https://app.viam.com/) remote control interface.
 
-Viam natively supports a wheeled base model, but if you have a quadruped or other form of rover that isn't compatible with the wheeled model, you have two options:
+Viam natively supports a wheeled base model, but if you have a quadruped or other form of rover that isn't compatible with the wheeled model, you can [create a custom component as a modular resource](/extend/modular-resources/).

```

Comment:
```suggestion
Viam natively supports a wheeled base model, but if you have a quadruped or other form of base that requires a different underlying implementation, you can [create a custom component as a modular resource](/extend/modular-resources/).

```

As it could be boat, drone, etc